pages/task/list/views/tasklist.jsx

```diff
@@ -64,9 +64,10 @@ function TaskTabs({ tabs, selected, hasTasks }) {
 
 export default function Tasklist() {
   const { workflowConnectionError, isAsruUser, progressOptions = [] } = useSelector(state => state.static);
-  const taskCount = useSelector(state => state.datatable.pagination.totalCount);
-  const progress = useSelector(state => state.static.progress) || progressOptions[0];
-  const hasTasks = taskCount > 0;
+  const { filters, schema, pagination } = useSelector(state => state.datatable);
+  const progress = get(filters, 'active.progress[0]') || progressOptions[0];
+  const hasTasks = pagination.totalCount > 0;
+  schema.updatedAt.label = (progress === 'completed') ? 'Completed' : undefined;
```

Comment:
This line modifies the datatable schema directly, not sure if this is a "proper" use of react...